plugins/kotlin/src/main/java/org/vineflower/kotlin/KotlinChooser.java

```diff
@@ -43,7 +43,7 @@ public boolean isLanguage(StructClass cl) {
         StructAnnotationAttribute attr = cl.getAttribute((Key<StructAnnotationAttribute>) key);
         for (AnnotationExprent anno : attr.getAnnotations()) {
           if (anno.getClassName().equals("kotlin/Metadata")) {
-            setContextVariables(cl);
+//            setContextVariables(cl);
```

Comment:
Was there a specific reason this is commented out? A comment here would be good.